notebooks/EMUKIT-Max-Value-Entropy-Search-Example.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<p>Can you proofread this intro a bit? Just things like typos and such.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/239/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Sorted!